File: c programming/practice set/c programing/chapter_3/problem_2.c
/*
write a program to determine wheather a student has passed or failed . 
to pass a student required a total of 40% and at least 33% in each subject.
Take marks as input from user!
*/
#include<stdio.h>
int main(){
    int bengali, english, math, total_marks;
    printf("Enter the marks in begali:");
    scanf("%d",& bengali);
    printf("Enter the marks in english:");
    scanf("%d",& english);
    printf("Enter the marks in math:");
    scanf("%d",& math);
    total_marks=((bengali+english+math)/3);

    if(bengali<33 || english<33 || math<33){
        printf("You have failed!");
    }else if(total_marks<40){
        printf("You have failed in the exam due to low percentage!");
   
    }else{
        printf("You have passed the exam!");
    }
    return 0;

}

File: Python/chapter6/problem2.py
'''Write a program to find out whether a student has passed or failed if it requires a 
total of 40% and at least 33% in each subject to pass. Assume 3 subjects and 
take marks as an input from the user. '''

b=int(input("Enter the marks in bengali:"))
e=int(input("Enter the marks in english:"))
m=int(input("Enter the marks in math :"))

total_marks=(b+e+m)/3

if(total_marks>=40 and b>=33 and e>=33 and m>=33 ):
    print("You have passed in the exam!")
else:
    print("You have failed !")
    